pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -16,6 +16,22 @@ def is_large_tensor(tensor, threshold):
     return size > threshold
 
 
+def _is_stripped(tensor):
+    if len(tensor.external_data) == 0:
+        return False
```

Comment:
I think this condition can now be removed.
```suggestion
```

Reply:
fixed